documentation/reference/qfieldcloud/jobs.en.md

```diff
@@ -81,7 +86,37 @@ A `delta_apply` job might result in `FAILED` status. Check the non-exhaustive li
 - The project is too big and the job has failed to run.
 - There are hidden files and directories within the project that are preventing the normal work of QFieldCloud. Hidden files and directories are those starting with a leading dot (`.`).
 
-### Re-apply changes in QFieldCloud
+##### Understanding conflicts `delta_apply` jobs
+
+Conflicts occur when any of these conditions is met:
+
+1) distinct users set the same attribute on the same feature to different values starting from the same old value;
+2) a primary key is employed twice.
+
+While highly unlikely have conflicts, preventing, mitigating and resolving conflicts is important to maintain data integrity in a healthy QGIS project. Here are some tips and tricks to do just that.
+
+Tips to avoid conflicts:
+
+- For updating existing features based on field conditions, plan and designate the features each user will update.
+- Users should not change the value of the primary key attribute.
+- Use a truly unique primary key, such as UUID (`uuid()`) or current timestamp (`epoch(now())`).
```

Comment:
Should we recommend one option only (uuid)?

Reply:
```suggestion
- Use a truly unique primary key, such as UUID (`uuid()`).
```
the current TS I guess was added for people forced to use a numeric ID.